src/commander/index.ts

```diff
@@ -26,8 +26,8 @@ async function main() {
 
   // Define menu options for client selection
   const menus = [
-    { name: 'synchronizer', value: 'synchronizer' },
-    { name: 'validator', value: 'validator' },
+    { name: 'Synchronizer', value: 'synchronizer' },
```

Comment:
use enumeration.ts

Reply:
Change to Client.Synchronizer and Client.Validator